prediction_market_agent/agents/arbitrage_agent/data_models.py

```diff
@@ -0,0 +1,70 @@
+import typing as t
+
+from prediction_market_agent_tooling.markets.agent_market import AgentMarket
+from pydantic import BaseModel, computed_field
+
+
+class Correlation(BaseModel):
+    correlation: float
```

Comment:
would be good to add a `reasoning` field so we can debug from the langfuse traces if it's not behaving as expected.